### ISSUE TITLE
Developing products with a design system/component-based approach

### ISSUE THREAD

**Issue by marhigh** [mainmatter/playbook#99]
Currently, it is not 100% clear that a design system is not only a deliverable of the design department (a reusable UI library in pixels).

Should we detail what a design system is on the technical side? What are the important points to bring up that ensure the system is flexible, maintainable, stable, scalable, and successful? What is important in a UI viewer? How to make sure the system is not abandoned?

**Comment by marcoow** [comment]
Yes, those are all good points. Do you have any concrete guidelines in mind and could prepare a PR?
